### PR TITLE
Added paypal,slack,stripe,wepay and yahoo providers

### PR DESCRIPTION
**Pull request by rakesh-eltropy**
Hi Mark,
I have added Paypal, Slack, Stripe, Wepay and Yahoo providers. Kindly review and merge in the master.

Thanks,
Rakesh Goyal

### CODE REVIEW

File: gothic/gothic.go [hunk 1]
```diff
@@ -159,7 +159,18 @@ var CompleteUserAuth = func(res http.ResponseWriter, req *http.Request) (goth.Us
 		return goth.User{}, err
 	}
 
-	return provider.FetchUser(sess)
+	user, err := provider.FetchUser(sess)
+	//Added to test if refresh token implementation is working properly or not
```
Comment (markbates) [inline gothic/gothic.go]: This probably shouldn't be in `gothic`. `gothic` is meant to be used in applications, not a place to test things. If this was here people would always be refreshing tokens in production when they don't mean to be.

**Comment by markbates** [comment]
Wow! Another amazing PR!! Thank you so much.

**Comment by rakesh-eltropy** [comment]
Thanks Mark! Goth has made my life so much easier. Happy to contribute back.

**Comment by zmb3** [comment]
Nice work Rakesh. You're on a roll!

**Pull request merged**